release-calendar/src/client/models/release-channel.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Channel, Release} from '../../types';
+
+function getMostRecent(channel: Channel, releases: Release[]): string {
```

Comment:
Say there are 50 releases. This iterates through it for every channel. Is there another way to get current releases that might be more performant?

Reply:
Can i sort them first and then use `.find()` ? 50*log(50) = 85ish.